spk/beets/Makefile

```diff
@@ -1,16 +1,17 @@
 SPK_NAME = beets
-SPK_VERS = 1.4.7
-SPK_REV = 3
+SPK_VERS = 1.4.9
+SPK_REV = 4
 SPK_ICON = src/beets.png
 
+PIP = pip3
 WHEELS = src/requirements.txt
-SPK_DEPENDS = "python>=2.7.8-11"
+SPK_DEPENDS = "python3>=3.5.6-8"
 
 MAINTAINER = ymartin59
-DESCRIPTION = "Beets is the media library management system for obsessive-compulsive music geeks. The purpose of beets is to get your music collection right once and for all. It catalogs your collection, automatically improving its metadata as it goes. It then provides a bouquet of tools for manipulating and accessing your music. ReplayGain plugin is not available because of lacking dependencies."
+DESCRIPTION = "Beets is the media library management system for obsessive-compulsive music geeks. The purpose of beets is to get your music collection right once and for all. It catalogs your collection, automatically improving its metadata as it goes. It then provides a bouquet of tools for manipulating and accessing your music. Plugins not available due lacking dependencies: AcousticBrainz Submit, Gmusic, ReplayGain."
```

Comment:
nit: Missing 'to' in: "Plugins not available due to lacking dependencies:"

Reply:
too short sentence ;-)